projects/components/src/table/table.component.ts

```diff
@@ -226,7 +227,7 @@ export class TableComponent
   public filters?: TableFilter[];
 
   @Input()
-  public mode?: TableMode = TableMode.Flat;
+  public mode: TableMode = TableMode.Flat;
```

Comment:
nit: Inputs should always be optional to support things like an async pipe which will explicitly override a default with null.